### PR TITLE
FEAT: Like, Dislike videos

### PR DESCRIPTION
**Pull request by Ayushi20-19**
image

Liked videos display page

image

<h1>FEATURE ADDED</h1>
<ul>
<li>data context and reducer</li>
<li>user can like Video</li>
<li>user can dislike video from gallery and also from liked page</li>
</ul>

note - login before liking videos

### CODE REVIEW

**Review by Veekshith-Thoram — APPROVED** [review]

File: src/context/data-context.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      dataDispatch({
+        type: "GET_VIDEOS",
+        payload: response.data.videos,
+      });
+    } else {
+      throw new Error("Error in data fetch");
+    }
+  }, []);
+
+  console.log(dataState.videos);
```
Comment (mohammed-sas) [inline src/context/data-context.js]: Remove the logs

File: src/services/likes/getLikesDataService.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const getLikesDataService = (localToken) => {
+  return axios.get("/api/user/likes", {
```
Comment (mohammed-sas) [inline src/services/likes/getLikesDataService.js:4]: Check response status before returning

File: src/services/videosData/getVideosService.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const getVideosService = () => {
+  return axios.get("/api/videos");
```
Comment (mohammed-sas) [inline src/services/videosData/getVideosService.js:4]: Check response status

**Review by mohammed-sas — APPROVED** [review]
Ui looks great
Files are well organised
Other than suggested in review your pr is good to merge

**Pull request merged**